.moban.d/travis.yml

```diff
@@ -1,18 +1,17 @@
 {% extends 'travis.yml.jj2' %}
 
 {%block extra_matrix %}
-matrix:
-  include:
-    - python: 2.7
-      env: MINREQ=1
+env:
+  - MINREQ=0
+  - MINREQ=1
 {%endblock%}
 
 {%block custom_python_versions%}
 python:
   - &pypy2 pypy2.7-6.0
-  - 3.8-dev
```

Comment:
Yaml fixed upstream. This change can be removed if you want

Reply:
let's keep it. it does find problem in ruamel.yaml, doesn't it?

Reply:
Moved to https://github.com/moremoban/moban/issues/289